### PR TITLE
Edit group profile

### PR DESCRIPTION
**Pull request by HSalmon**

### CODE REVIEW

File: app/services/group_service.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@ class GroupService
...
     actor.ability.authorize! :create, group
-    group.save! && group.mark_as_setup! && group.add_admin!(actor)
+    group.save && group.mark_as_setup! && group.add_admin!(actor)
     group
   end
 
   def self.update(group:, params:, actor:)
     actor.ability.authorize! :update, group
-    group.update! params
+    group.update params
```
Comment (gdpelican) [inline app/services/group_service.rb]: Ah, yep, that'll do it.

File: lineman/app/components/edit_group_form/edit_group_form.coffee [hunk 2]
```diff
@@ -6,9 +6,14 @@ angular.module('loomioApp').factory 'EditGroupForm', ->
     $scope.submit = ->
       $scope.group.save().then ->
         $scope.$close()
-        FlashService.success 'group_form.edit_group_success'
-      , ->
-        $rootScope.$broadcast 'pageError', 'cantEditGroup', $scope.group
+        FlashService.success 'edit_group_form.messages.success'
```
Comment (gdpelican) [inline lineman/app/components/edit_group_form/edit_group_form.coffee]: Time to start thinking about how we're going to generalize this pattern, as I have a feeling we're about to implement 10x over the next week or two.

I was thinking
```
(response) ->
  ErrorService.applyError response, $scope.group
```
```
ErrorService ($rootScope) ->
  applyError: (response, model) ->
    case response.status
      when 422 then $scope.group.setErrors response.data.errors
      when 400 then @broadcastError 'badRequest', response
      when 401, 403 then @broadcastError 'requestForbidden', response
      when 404 then @broadcastError 'connectivityLost', response
      when 500 then @broadcastError 'serverError', response
    
  broadcastError: (errorType, response) ->
    $rootScope.$broadcast errorType, response
```

Comment (HSalmon) [inline lineman/app/components/edit_group_form/edit_group_form.coffee]: @gdpelican, yeah it seems pretty necessary. Think it's a separate ticket though - maybe you wanna stick one in the sprint board with your suggestion?

Comment (gdpelican) [inline lineman/app/components/edit_group_form/edit_group_form.coffee]: Yeah I'll do that.

**Pull request merged**